vcr/api/oidc4vci/v0/api.go

```diff
@@ -119,18 +115,46 @@ func (w Wrapper) Routes(router core.EchoRouter) {
 	}))
 }
 
+func (w Wrapper) getWallet(ctx context.Context, holderDIDAsString string) (holder.OIDCWallet, error) {
```

Comment:
method is used in other file so expect it to be there. 
`holderDIDAsString string` is a bit overkill? `holder string` is enough info for anybody to know they should pass the string format?
The method name is non-descriptive. Also due to the `holder.OIDCWallet` naming. Would `getHandler(..) (holder.OpenIDHandler, error)` be more clear?

Reply:
I agree `OIDCWallet` is a lame term, `Wallet` would be nicer to read. `Handler` or `Wallet`; i.m.o. a holder has a wallet, thus a `holder.Wallet` type sounds logical. But the wallet is OpenID4VCI specific. So then `OpenIDHandler` might be not so bad after all (and the `holder` can have a more high-level `Wallet` type in future.